src/lib/Form/Factory/FormFactory.php

```diff
@@ -1018,18 +1023,22 @@ public function forgotUserPasswordWithLogin(
     /**
      * @param \EzSystems\EzPlatformAdminUi\Form\Data\User\UserPasswordResetData $data
      * @param null|string $name
```

Comment:
You might as well fix this: 
```suggestion
      * @param string|null $name
```